frontend/app/routes/$lang+/_protected+/personal-information+/phone-number+/confirm.tsx

```diff
@@ -42,24 +42,17 @@ export async function loader({ request }: LoaderFunctionArgs) {
   await raoidcService.handleSessionValidation(request);
 
   const userId = await userService.getUserId();
-
-  if (!userId) {
-    instrumentationService.countHttpStatus('phone-number.confirm', 302);
-    return redirectWithLocale(request, '/');
-  }
-
   const userInfo = await userService.getUserInfo(userId);
   if (!userInfo) {
-    instrumentationService.countHttpStatus('phone-number.confirm', 302);
-    return redirectWithLocale(request, '/');
+    instrumentationService.countHttpStatus('home-address.edit', 404);
```

Comment:
Copy/paste error. The metric name here should be:

```suggestion
    instrumentationService.countHttpStatus('phone-number.confirm', 404);
```